src/org/omegat/filters3/xml/xliff/XLIFFOptions.java

```diff
@@ -55,6 +55,8 @@
     private static final String OPTION_FORCE_SHORTCUT_2_F = "forceshortcut2f";
     private static final String OPTION_IGNORE_TYPE_4_PH_TAGS = "ignoretype4phtags";
     private static final String OPTION_IGNORE_TYPE_4_BPT_TAGS = "ignoretype4bpttags";
+    private static final String OPTION_STATE_KEEP_NEEDS = "statekeepneeds";
```

Comment:
I find `KEEP_NEEDS` to be a bit cryptic. Could you make it `KEEP_NEEDS_TRANSLATION`?

If that's too long for your taste, it could be `KEEP_NEEDS_T9N`

Please expand the method names to match.